CommonApplication.cmake

```diff
@@ -17,7 +17,8 @@
 # * NAME_OMIT_CHECK_TARGETS do not create cppcheck targets
 # * ARGN for optional add_executable parameters
 # * NAME_DATA files for share/Project/data (in binary and install dir)
-# * NAME_ICON optional .icns file (Mac OS GUI applications only)
+# * NAME_DESKTOP optional .desktop file (Linux GUI applications only)
+# * NAME_ICON optional .icns (Mac OS) or .png (Linux) file (GUI app. only)
```

Comment:
The name should be without extension, so user CMake code is simpler. We can append the extension here and document it.